index.html

```diff
@@ -0,0 +1,127 @@
+              laid off due to budget cuts. Inspired by Elvin Charles Stakman, he
+              returns to school study under Stakman, who teaches him about
+              breeding pest-resistent plants.
+            </li>
+            <li>
+              <span>1941</span> - Tries to enroll in the military after the
+              Pearl Harbor attack, but is rejected. Instead, the military asked
+              his lab to work on waterproof glue, DDT to control malaria,
+              disinfectants, and other applied science.
+            </li>
+            <li>
+              <span>1942</span> - Receives a Ph.D. in Genetics and Plant
+              Pathology
+            </li>
+            <li>
+              <span>1944</span> - Rejects a 100% salary increase from Dupont,
+              leaves behind his pregnant wife, and flies to Mexico to head a new
+              plant pathology program. Over the next 16 years, his team breeds
+              6,000 different strains of disease resistent wheat - including
+              different varieties for each major climate on Earth.
+            </li>
+            <li>
+              <span>1945</span> - Discovers a way to grown wheat twice each
+              season, doubling wheat yields
+            </li>
+            <li>
+              <span>1953</span> - crosses a short, sturdy dwarf breed of wheat
+              with a high-yeidling American breed, creating a strain that
+              responds well to fertilizer. It goes on to provide 95% of Mexico's
+              wheat.
+            </li>
+            <li>
+              <span>1962</span> - Visits Delhi and brings his high-yielding
+              strains of wheat to the Indian subcontinent in time to help
+              mitigate mass starvation due to a rapidly expanding population
+            </li>
+            <li><span>1970</span> - receives the Nobel Peace Prize</li>
+            <li>
+              <span>1983</span> - helps seven African countries dramatically
+              increase their maize and sorghum yields
+            </li>
+            <li>
+              <span>1984</span> - becomes a distinguished professor at Texas A&M
+              University
+            </li>
+          </ul>
+        </div>
+      </div>
+      <div class="info-section">
+        <p>
+          If you want to know more about Dr. Borlaug please
+          <a
+            id="tribute-link"
+            class="link"
+            href="https://en.wikipedia.org/wiki/Norman_Borlaug"
+            target="_blank"
+            >Click here</a
+          >
+        </p>
+      </div>
+      >
+    </main>
+    <!--
+    
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+    <script src="js/index.js"></script>
```

Comment:
- [ ] Please be more polite, professional, friendly and transparent in your comments while doing code reviews. Your comment looks like you are ordering.

## Your comment

> You are missing a DOCTYPE declaration and a lang attribute which specifies the language of the document. Please consider using them as it is considered a good practice

## Comment should be like

> Please remove the unnecessary `script` file from your `HTML` file, because having a script tag in your `HTML` looks for the file after the `website-paint` process and it will give the error in `console` which looks bad and make your project work unprofessional. Kindly make sure that there should be no error in your `production` ready code.